qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -553,7 +570,8 @@ def write(self, cmd):
 
         nr_bytes_written, ret_code = self.visa_handle.write(cmd)
         self.check_error(ret_code)
-        self._write_response = self.visa_handle.read()
+        if not fast:
```

Comment:
What happens to the response if `fast`? Isn't it still in some output queue of the QDac?

Reply:
I think this in general is a bad idea but they wanted it. Hence off by default and the warning in the doc string